src/main/java/net/snowflake/client/core/SFArrowResultSet.java

```diff
@@ -31,7 +31,8 @@
 
 /** Arrow result set implementation */
 public class SFArrowResultSet extends SFBaseResultSet implements DataConversionContext {
-  static final SFLogger logger = SFLoggerFactory.getLogger(SFArrowResultSet.class);
+  private static final SFLogger logger = SFLoggerFactory.getLogger(SFArrowResultSet.class);
+  private static final ObjectMapper OBJECT_MAPPER = ObjectMapperFactory.getObjectMapper();
```

Comment:
private static final field for logger has different convention than OBJECT_MAPPER - we should keep and use one of them